sys-config/ltoize/files/package.cflags/ltoworkarounds.conf

```diff
@@ -123,6 +177,7 @@ dev-lisp/clisp *FLAGS+=-falign-functions # Issue #128.  Function alignment probl
 # BEGIN: Deliberate -O3 workarounds
 media-libs/flac *FLAGS+=-fno-ipa-cp-clone # -fipa-cp-clone is enabled by default at -O3, breaks 'flac -d' if used in conjunction with -flto under GCC 8.3.0
 media-libs/faad2 *FLAGS+=-fno-tree-loop-vectorize # causes subtly wrong decoding
+>=x11-wm/mutter-3.28 /-O3/-O2 # maybe-uninitialized error in cogl-pipeline-layer-state.c
```

Comment:
No issues here, at least during compilation

Reply:
Still reproducible with `=x11-wm/mutter-3.30.2-r2` for me.
[build.log](https://github.com/InBetweenNames/gentooLTO/files/3343449/build.log)

Reply:
Interesting.  I've observed this on two systems now.  The `may be uninitialized` error is a warning that is being treated as an error, but it seems it only is treated like that on some systems and not others.  How was your gcc built?

Reply:
Can't upload the log for whatever reason, linking as gist. [gcc-9.1.0-r1:20190601-212108.log](https://gist.github.com/petronio/abd38697f19385cebd6dd3eca9d80294)

Reply:
Are you injecting `BOOT_CFLAGS` at all during the compile?

Reply:
Nope, and `override-flagomatic` is disabled as well.

Reply:
I noticed you disabled LTO on this package -- could you try it with LTO and see if it works?  I noticed this happen to one other user too on a different package.

Reply:
Building with LTO gets the sed issue, so that prevents further testing.

Reply:
I see -- so it seems like we need to figure out what's going on in these configure scripts on your system.  Do you have a non-hardened gcc you can test with?

Reply:
I don't. Although I might be able to build 8.3.0 since it's slotted, but that probably wouldn't be a good test.

Reply:
I'll try building a hardened GCC next to see if I can reproduce it that way.

Reply:
It may be easier for you to combine the `default/linux/amd64/17.1/desktop/gnome/systemd` and `default/linux/amd64/17.1/hardened` hardened profiles in a local profile, as I have, then emerge world.

Reply:
I haven't rebuilt GCC yet, but I just realized that the hardened GCC might be forcing `-Werror` on you, and there are more warnings in GCC 9.1 than there were in previous versions.  I'll try to dig around the Gentoo hardening patches to find anything with that effect.

Reply:
> I haven't rebuilt GCC yet, but I just realized that the hardened GCC might be forcing `-Werror` on you, and there are more warnings in GCC 9.1 than there were in previous versions. I'll try to dig around the Gentoo hardening patches to find anything with that effect.

Hardened doesn’t force Werror. But there are more things that might be flagged as errors in gcc 9 I’d guess.